django/contrib/auth/management/commands/changepassword.py

```diff
@@ -14,7 +14,7 @@ class Command(BaseCommand):
     requires_migrations_checks = True
     requires_system_checks = False
 
-    def _get_pass(self, prompt="Password: "):
+    def _get_pass(self, prompt="Password: "):  # pragma: no cover
```

Comment:
I think it would be more useful to mock `getpass.getpass` than to skip testing this entirely.